src/encoded/static/components/static-pages/components/internal/SubmissionStatus.js

```diff
@@ -0,0 +1,604 @@
+        const payload = {
+            limit: PAGE_SIZE,
+            from: this.state.tablePage * PAGE_SIZE,
+            filter: this.state.filter,
+        };
+
+        ajax.load(
+            '/get_submission_status/',
+            (resp) => {
+                if(resp.error){
+                    console.error(resp.error);
+                    return;
+                }
+                this.setState({
+                    initialLoading: false,
+                    loading: false,
+                    fileSets: resp.file_sets,
+                    numTotalFileSets: resp.total_filesets,
+                });
+            },
+            'POST',
+            fallbackCallback,
+            JSON.stringify(payload)
+        );
+    }
+
+    componentDidMount() {
+        this.getSubmissionCenters();
+        this.getData();
+    }
+
+    setFilter = (filterName, selection) => {
+        const filter = this.state.filter;
+        filter[filterName] = selection;
+        this.applyFilter(filter);
+    };
+
+    applyFilter(filter) {
+        this.setState(
+            (prevState) => ({
+                filter: filter,
+                tablePage: 0,
+                loading: true,
+            }),
+            function () {
+                this.getData();
+            }
+        );
+    }
+
+    getSubmissionCenterSelect() {
+        if (this.state.submission_centers == 0) {
+            return (
+                <React.Fragment>
+                    <select className="custom-select" defaultValue="all">
+                        <option value="all">All</option>
+                    </select>
+                </React.Fragment>
+            );
+        } else {
+            const options = [
+                <option value="all">All</option>,
+                <option value="all_gcc">All GCCs</option>,
+            ];
+            this.state.submission_centers.forEach((sc) => {
+                options.push(<option value={sc.title}>{sc.title}</option>);
+            });
+            return (
+                <React.Fragment>
+                    <select
+                        className="custom-select"
+                        defaultValue="all_gcc"
+                        onChange={(e) =>
+                            this.setFilter('submission_center', e.target.value)
+                        }>
+                        {options}
+                    </select>
+                </React.Fragment>
+            );
+        }
+    }
+
+    getFilesetStatusSelect = () => {
+        return (
+            <React.Fragment>
+                <select
+                    className="custom-select"
+                    defaultValue="in review"
+                    onChange={(e) =>
+                        this.setFilter('fileset_status', e.target.value)
+                    }>
+                    <option value="all">All</option>
+                    <option value="in review">In Review</option>
+                    <option value="released">Released</option>
```

Comment:
You probably want to include "public" and "restricted" - it may be easier to resolve the possible values from a facet in the backend and just allow all of them. You can get this from a standard (limit=0) search on files.